README.md

```diff
@@ -1,15 +1,12 @@
 # Whale-spotting
 
-# Setup
-Run the command `SET ASPNETCORE_Environment=Development` in your terminal.
```

Comment:
is this not required any more?

Reply:
I'm not too sure what has happened here as we have not edited the Read.Me